src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -584,4 +584,18 @@ public function testPrivateWithIgnoreOnInvalidReference()
         $container = new \Symfony_DI_PhpDumper_Test_Private_With_Ignore_On_Invalid_Reference();
         $this->assertInstanceOf('BazClass', $container->get('bar')->getBaz());
     }
+
+    public function testArrayParameters()
+    {
+        $container = new ContainerBuilder();
+        $container->setParameter('array_1', array(123));
+        $container->setParameter('array_2', array(__DIR__));
+        $container->register('bar', 'BarClass')
+            ->addMethodCall('setBaz', array('%array_1%', '%array_2%', '%%string%%'));
```

Comment:
please make the content between the `%%` match the name of an existing parameter though (to ensure that working properly is not just due to the `hasParameter` check)